app/widget/timeruler/seekablewidget.cpp

```diff
@@ -44,6 +44,10 @@ SeekableWidget::SeekableWidget(QWidget* parent) :
   playhead_width_ = QtUtils::QFontMetricsWidth(fm, "H");
 
   setContextMenuPolicy(Qt::CustomContextMenu);
+
+  marker_layout_ = new QHBoxLayout(this);
```

Comment:
Wouldn't bother with the QHBoxLayout, I think you'll most likely have to manually control their positions anyway